### ISSUE TITLE
Google Auth login: This browser or app may not be secure.

### ISSUE THREAD

**Issue by mahalde** [brave/brave-browser#8544]
## Description 

Whenever I try to login via Google, the login fails with the following message:

**Couldn't sign you in**
This browser or app may not be secure.

## Steps to Reproduce

   1. Go to google.com
   2. Click the login button in the top right corner.
   3. Log in with your information

## Actual result:

It does not log you in, instead shows the login error.
![image](https://user-images.githubusercontent.com/39100405/75959839-4b1a4a00-5ebf-11ea-8b66-b53c0903437e.png)

## Expected result:
It should log me in.

## Reproduces how often: 

Every time.

## Brave version (brave://version info)

Brave | 1.4.96 Chromium: 80.0.3987.132 (Official Build) (64-bit)
-- | --
Revision | fcea73228632975e052eb90fcf6cd1752d3b42b4-refs/branch-heads/3987@{#974}
OS | Windows 10 OS Version 1809 (Build 17763.1039)

## Version/Channel Information:

- Can you reproduce this issue with the current release?
Yes
- Can you reproduce this issue with the beta channel?
Haven't tried
- Can you reproduce this issue with the dev channel?
Haven't tried
- Can you reproduce this issue with the nightly channel?
Haven't tried

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields?
No.
- Does the issue resolve itself when disabling Brave Rewards?
No.
- Is the issue reproducible on the latest version of Chrome? 
No.

## Miscellaneous Information:

It only started to happen after I cleared my browser data. I even tried uninstalling the browser and reinstalling it, without any changes.

**Comment by C-Radius** [comment]
Same thing happening to me.

**Comment by Tommos0** [comment]
This just started happening here too

**Comment by rebron** [comment]
@mahalde @C-Radius @Tommos0 Are you seeing a javascript block icon in the url bar? Once that's allowed, should be working yes?

@kjozwiak @jumde  I was able to reproduce this on 1.7.98, 
Just go to google.com, hit sign-in button. Enter username. And then I see the error above. It looks like we're blocking javascript on Google.com as I'm seeing the javascript block icon in the url bar. That shouldn't be there.

Not seeing this same behavior in 1.8.x though with a new profile.

**Comment by rebron** [comment]
@mahalde @C-Radius @Tommos0. Can you take a look at one more thing? In shields settings, can you toggle on/off Scripts blocked in the advanced panel and see if this is working for you now.

**Comment by mahalde** [comment]
When I had the issue, I had Shields turned off and also enabled JavaScript. I did not toggle on/off Scripts blocked in the advanced panel.

Funny enough the issue resolved when I logged into google on my home network, not on the corporate network where the issue happened.

Right now I cannot reproduce the issue, but I hope my information helps!

**Comment by rebron** [comment]
Closing per last comment. Thanks for following up @mahalde

**Comment by al3xmaat** [comment]
I'm currently experiencing this issue and toggleing on/off Scripts blocks didn't resolve it for me.

**Comment by theblazehen** [comment]
Another way is to select the "Don't know username" option, to initiate the account recovery process which takes you to a different login page which works

**Comment by mahalde** [comment]
Thought I give an update on this situation, I installed Brave on a new device and it started happening again. I was googling for a solution and found this [stackoverflow post](https://stackoverflow.com/a/59673634/10207459) which did the trick for me.

### TL;DR: **Turn off remote debugging**

**Comment by Ecif** [comment]
Thanks! Had to remove --remote-debugging-port=9222 from the target also.
